src/appshell/qml/AppMenuBar.qml

```diff
@@ -52,6 +53,8 @@ ListView {
     NavigationPanel {
         id: navPanel
         name: "AppMenuBar"
+        enabled: root.enabled && root.visible
+        accessible.name: qsTrc("appshell", "App menu bar") + " " + navPanel.directionInfo
```

Comment:
Including `navPanel.directionInfo` in the name is helpful at first but I think blind users will quickly get tired of hearing _"direction is horizontal"_ or _"direction is vertical"_ all the time. Instead, I think we should adopt the following convention:

- Toolbars and menu bars are always __horizontal__.
- Item views (e.g. tables and trees) are always __both directions__ unless there is only one column of data in the model.
- Everything else (e.g. menu items, lists, single-column tables and trees) are always __vertical__.

This should make it unnecessary to say the direction explicitly in most cases. If there are specific places where we need to break the convention then we can always say it in those places only.

@igorkorsukov, what do you think?

Reply:
Yes, I agree with that.
During development, I added direction information for demonstration and debugging purposes only.
Now we need to think and solve all the moments as they should be in production.

Reply:
thanks, removed